clab/file.go

```diff
@@ -132,19 +132,20 @@ func CreateDirectory(path string, perm os.FileMode) {
 	}
 }
 
-// CreateNodeDirStructure create the directory structure and files for the clab
+// CreateNodeDirStructure create the directory structure and files for the lab nodes
 func (c *cLab) CreateNodeDirStructure(node *Node) (err error) {
 	c.m.RLock()
 	defer c.m.RUnlock()
+
+	// create node directory in the lab directory
+	CreateDirectory(node.LabDir, 0777)
```

Comment:
we don't need to create directories for bridge, alpine and linux kinds, so maybe keep this under `case "srl"` and `case "ceos"` only ?

Reply:
we will have more NOS kinds which will require some configs or files to be mounted.
Lets maybe make the creation conditional if kind is not `linux | bridge`

Reply:
that works too

Reply:
added this in c508c96

I want to create another PR for removing alpine kind, do you agree?

Reply:
you already removed it from the dir structure func, no ?